src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl

```diff
@@ -4483,6 +4483,52 @@ unit:CFU
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "Colony Forming Unit"@en .
 
+  unit:CFU-PER-GM
+  a qudt:Unit ;
+  dcterms:description "The count of \"Colony Forming Units\" in a gram, used for solid materials"^^qudt:LatexString ;
+  qudt:applicableSystem sou:ASU ;
+  qudt:applicableSystem sou:CGS ;
+  qudt:applicableSystem sou:CGS-EMU ;
+  qudt:applicableSystem sou:CGS-ESU ;
+  qudt:applicableSystem sou:CGS-GAUSS ;
+  qudt:applicableSystem sou:IMPERIAL ;
+  qudt:applicableSystem sou:PLANCK ;
+  qudt:applicableSystem sou:SI ;
+  qudt:conversionMultiplier 1000 ;
+  qudt:conversionMultiplierSN 1.0E3 ;
+  qudt:hasDimensionVector qkdv:A0E0L0I0M-1H0T0D1 ;
```

Comment:
@steveraysteveray @jhodgesatmb @ralphtq I believe a dimension vector can only have D1 if all other dimensions are 0. right?

Reply:
Correct.

Reply:
I guess it has to be qudt:hasDimensionVector `qkdv:A0E0L0I0M-1H0T0D0` then.

Reply:
Yes, for the -PER-GM one, and similar correction for the -PER-MilliL one.